dev-docs/analytics/smartyads.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: analytics
+title: Smartyads
+description: Smartyads Analytics Adapter
+modulecode: smartyads
+---
+
```

Comment:
Do you want to add any additional information? Like what your product does and how to get in touch with you?

Reply:
@muuki88 no, it's not necessary